docs/developer_guide/web_app_guide.md

```diff
@@ -0,0 +1,188 @@
+
+1. Create a new folder for the project (e.g. `mkdir ~/monero-javascript_web_app`)
+2. Enter the new directory (`cd ~/monero-javascript_web_app`)
+3. Download and run the script:
+
+`$ bash <(curl -sL https://raw.githubusercontent.com/woodser/xmr-sample-app/master/web_template_script.sh)`
+
+<!--
+#### Windows
+1. Create a new folder for the project (e.g. `mkdir C:\Users\[username]\monero_javascript_web_app`)
+2. Enter the new directory (`cd C:\Users\[username]\monero_javascript_web_app`)
+3. Download and run the script:
+```
+powershell -Command "(new-object net.webclient).DownloadString('https://raw.githubusercontent.com/woodser/xmr-sample-app/add_starter_scripts/webTemplateScript.bat')"
+webTemplateScript.bat
+```
+-->
+Alternately, you can manually download and run the shell scripts.
+[Linux/Mac](https://github.com/timetherewere/monero-javascript/raw/include_compressed_template/web_template_script)
+<!--
+[Windows](https://github.com/timetherewere/monero-javascript/raw/include_compressed_template/web_template_script.bat)
+-->
+The script will configure the project folder and serve the starter web application on port 9100. Open a web browser and type "localhost:9100" in the URL bar to view the application.
+
+_Note_:
+* Unlike create-react-app, the xmr-sample-app script will not automatically open a web browser. You must open the browser and navigate to "localhost:9100" manually.
+* The script will not automatically refresh the server and page whenever a source file is modified.
+*   In order for the server to reflect changes to source files, you need to stop it by pressing "ctrl-c" in the terminal where the server is running and and restart the server by typing
+`./bin/build_browser_app`.
+
+## Modifying the offline wallet generator to display in a browser
+
+The starter web application also serves as a demonstration of some basic monero-javascript methods (check the developer console to see the results) and a visual mock-up of a wallet application that a developer could potentially produce using the monero-javascript library.
+
+Although this application _does_ show off a pretty slick Monero wallet design, it is not useful for building an offline wallet generator, so remove the extraneous code and files to make room for your wallet generator.
+
+* Delete the sample wallet images:
+rm -rf src/mockups
+<!--
+### Windows
+rmdir /s /q src\mockups
+-->
+* Navigate to the "./src/" directory.
+* Modify the index.html file so that it displays a blank page and runs the javascript code:
+
+```
+<!DOCTYPE html>
+
+<html>
+  <script type="text/javascript" src="xmr-sample-app.js"></script>
+</html>
+```
+_Note that index.html references a javascript file called "xmr-sample-app.js" rather than index.html._ This is due to the fact that the build_browser_app shell script assembles the final web application image in the "browser_build" folder, where it saves the working copy of index.js under the file name "xmr-sample-app.js".
```

Comment:
`Note that index.html references a javascript file called "xmr-sample-app.js" rather than index.html.`
Should be `rather than index.js.`